setup.py

```diff
@@ -63,10 +63,7 @@ def _load_readme():
         "enum34 >= 1.0 ; python_version<'3.4'",
         "six >= 1.0 ; python_version<'3'",
     ],
-    setup_requires=[
-        "setuptools_scm",
-        "setuptools_scm_about",
-    ],
+    setup_requires=["setuptools_scm", "setuptools_scm_about"],
```

Comment:
`black` insisted on this one, although I believe it was `black` who formatted the previous setup.py as well.. Could there be version differences?